### ISSUE TITLE
Missing Private Modifier on mapping type _fundedMiners

### ISSUE THREAD

**Issue by bearcanrun** [Superalgos/ALGOToken#14]
Name of mapping type `_fundedMiners` suggests private variable so mapping should be designated as `private`.

```
# AlgoPool.sol

L20: mapping(address => bool) _fundedMiners;

Fix: mapping(address => bool) private _fundedMiners;

```

**Comment by pmmax** [comment]
Added a fix for this issue and other missing visibility modifiers: https://github.com/Superalgos/ALGOToken/pull/16

**Comment by bearcanrun** [comment]
Fixed by #16
